pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -307,7 +307,29 @@ protected void setupInitialSegments(String realtimeTableName, ZNRecord partition
 
       metadata.setCreationTime(now);
 
-      final long startOffset = getPartitionOffset(topicName, bootstrapHosts, initialOffset, i);
+      Map<String, String> decoderConfigs = null;
+      String consumerFactory = null;
```

Comment:
s/consumerFactory/consumerFactoryName

Otherwise the var name is confusing with the reference to the consumer factory.